tests/safeds/ml/classical/classification/test_support_vector_machine.py

```diff
@@ -11,17 +11,69 @@ def training_set() -> TaggedTable:
 
 class TestC:
     def test_should_be_passed_to_fitted_model(self, training_set: TaggedTable) -> None:
-        fitted_model = SupportVectorMachine(c=2).fit(training_set=training_set)
+        kernel = SupportVectorMachine.Kernel.Linear()
+        fitted_model = SupportVectorMachine(c=2, kernel=kernel).fit(training_set=training_set)
```

Comment:
Keep the Kernel as None. This method should only test the C Parameter and it would fail if something would be wrong with the Kernel implementation.

```suggestion
        fitted_model = SupportVectorMachine(c=2).fit(training_set=training_set)
```

Reply:
As discussed, it will be better to move the test methods to the TestKernel class and perform only the kernel checks. If not code-coverage won't reach a 100%.